src/components/select/select.scss

```diff
@@ -108,12 +108,17 @@ md-select {
     display: inline;
   }
 
-  *:first-child {
+
+  & > span:not(._md-select-icon) {
+    max-width: 100%;
     flex: 1 0 auto;
-    text-overflow: ellipsis;
-    white-space: nowrap;
-    overflow: hidden;
     transform: translate3d(0, 2px, 0);
+
+    ._md-text {
```

Comment:
Why is this nested **under**  `& > span:not(._md-select-icon)` ?
What is the use case ?